### PR TITLE
Favorite item

### PR DESCRIPTION
**Pull request by pdeguzman96**

### CODE REVIEW

File: app/web/__init__.py [hunk 1]
```diff
@@ -63,6 +64,7 @@ def register_blueprints(app):
     app.register_blueprint(address, url_prefix='/address')
     app.register_blueprint(country, url_prefix='/country')
     app.register_blueprint(address_type, url_prefix='/address_type')
+    app.register_blueprint(favorite_item, url_prefix='/favorite_item')
```
Comment (PansulBhatt) [inline app/web/__init__.py:67]: `/favorite_item` should be `/favorite-item`

File: app/web/api/__init__.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 address = Blueprint('address', __name__)
 address_type = Blueprint('address_type', __name__)
 country = Blueprint('country', __name__)
+favorite_item = Blueprint('favorite_item', __name__)
```
Comment (PansulBhatt) [inline app/web/api/__init__.py:8]: /favorite_item should be /favorite-item

File: app/web/api/favorite_item_routes.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
+from . import favorite_item
+from web.models.favorite_item import FavoriteItemTable
+from web.models.account import Account
+from web.api.api_utils import converter, exception_handler
+from datetime import date as d
```
Comment (PansulBhatt) [inline app/web/api/favorite_item_routes.py:5]: do we need this?

File: app/web/api/favorite_item_routes.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+from datetime import datetime as dt
+from web.database import db
+import json
+from flask import jsonify, request, current_app, abort
+
+@favorite_item.route('/', methods=['GET'])
+@exception_handler(custom_msg='Issue in fetching all favorite items')
+def get_all_favorite_items(return_json=True):
```
Comment (PansulBhatt) [inline app/web/api/favorite_item_routes.py:13]: Can you add some comments for this function?

File: app/web/api/favorite_item_routes.py [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        )
+    )
+    FavItemTbl = FavoriteItemTable(db_engine)
+    favitem_table_obj = FavItemTbl.table
+
+    item = memsql_session.query(favitem_table_obj).filter_by(id=favorite_item_id).one_or_none()
+    if not item:
+        abort(400, description=f'Favorite item with id {favorite_item_id} does not exist.')
+
+    db_engine.execute(favitem_table_obj.delete().where(favitem_table_obj.c.id == favorite_item_id))
```
Comment (PansulBhatt) [inline app/web/api/favorite_item_routes.py:96]: Probably add a comment that this is a hard delete

File: app/web/api/favorite_item_routes.py [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+    db.session.commit()
+    body['success'] = True
+    body['deleted_fav_item'] = favorite_item_id
```
Comment (PansulBhatt) [inline app/web/api/favorite_item_routes.py:100]: Don't need to return this

File: app/web/models/invoice_header.py [hunk 7]
```diff
@@ -1,6 +1,5 @@
 import os
-from sqlalchemy.types import Integer
```
Comment (PansulBhatt) [inline app/web/models/invoice_header.py:2]: Thanks for clearing this out

File: app/web/__init__.py (hunk 1 above)
Comment (pdeguzman96) [reply app/web/__init__.py:67]: Resolved, updating all of our routes for consistency

File: app/web/api/favorite_item_routes.py (hunk 3 above)
Comment (pdeguzman96) [reply app/web/api/favorite_item_routes.py:5]: no, removed

**Pull request merged**